### PR TITLE
hangman rota improved

### PR DESCRIPTION
**Pull request by gk-patel**
Small changes, 

- hangman hint fixed 
- rota gameover board

### CODE REVIEW

File: src/games/rota/Field.tsx [hunk 1]
```diff
@@ -22,18 +23,38 @@ const playerColors = {
...
     return (
-      <svg width="100%" height="100%" viewBox="-250 -250 500 500" className={css.RotaField} pointerEvents="visible">
-        <g strokeWidth="4" stroke="#cccccc">
-          {this._drawLinesAndCircle()}
-        </g>
-        <g>{this._placePieces()}</g>
-      </svg>
+      <div style={{ textAlign: 'center' }}>
+        <svg
+          width={`${this.props.boardSize}%`}
```
Comment (flamecoals) [inline src/games/rota/Field.tsx:28]: You have BOARD_SIZE and props.boardSize, what is the difference?

File: src/games/rota/board.tsx [hunk 2]
```diff
@@ -80,20 +80,33 @@ export class Board extends React.Component<IBoardProps, {}> {
     }
   }
 
+  _getBoard(boardSize = 100) {
```
Comment (flamecoals) [inline src/games/rota/board.tsx:83]: This seems to be an constant, so why make it a prop? Wouldn't it be better for it to just be a constant inside Field?

Comment (gk-patel) [reply src/games/rota/board.tsx:83]: nope. there is one call with 50.

File: src/games/rota/Field.tsx (hunk 1 above)
Comment (gk-patel) [reply src/games/rota/Field.tsx:28]: well, when the game gets over, it is called with 50% of the original size.

Comment (flamecoals) [reply src/games/rota/Field.tsx:28]: I see, can you rename one of those variables so they don't seem to mean the same thing?

You could rename the props to be boardSizePerc for percentage, or the other one for SVG_BOARD_SIZE

**Pull request merged**